components/Storyblok/SbHomepageMVP/SbHomepageMvp.tsx

```diff
@@ -39,33 +44,29 @@ export const SbHomepageMvp = ({
     ankle,
     mastheadPicker,
   },
-  blok,
 }: SbHomepageMvpProps) => {
   const ChangemakerCards = !!getNumBloks(changemakerCards) ? <CreateBloks blokSection={changemakerCards} /> : undefined;
   const ChangemakerCta = !!getNumBloks(changemakerCta) ? <CreateBloks blokSection={changemakerCta} /> : undefined;
 
   return (
-    <div {...storyblokEditable(blok)}>
+    <>
       <CreateStories stories={mastheadPicker} />
       <main id="main-content">
-        <div>
-          <Heading as="h1" srOnly>{title || 'Homepage'}</Heading>
```

Comment:
Deprecating this title field in Storyblok for screen reader only h1. SODA now recommend H1 matching the visual heading so no more custom mismatched visible and screen reader only heading.